src/components/header/MegaMenu/MegaMenu.tsx

```diff
@@ -19,7 +19,7 @@ export const MegaMenu = (
       <div className="grid-row grid-gap-4">
         {items.map((listItems, i) => (
           <div className="usa-col" key={`subnav_col_${i}`}>
-            <NavList items={listItems} type="megamenu" {...ulProps} />
+            <NavList items={listItems} megamenu={true} {...navListProps} />
```

Comment:
suggestion:  for `megamenu={true}` and I think I saw this one other place.  Leave off the attribute assignment when its `true.` Just by including the name of the attribute on the element it is by default true.